addons/website/views/snippets/s_product_list.xml

```diff
@@ -3,62 +3,94 @@
 
 <template name="Items" id="s_product_list">
     <t t-set="url" value="url or '#'"/>
-    <section class="s_product_list">
-        <div class="container-fluid">
+    <section class="s_product_list pt64 pb64" data-vcss="001">
+        <div class="container">
+            <h3 class="mb-4">Our finest selection</h3>
             <div class="row">
                 <div class="col-lg-2 col-md-4 col-6">
-                    <a t-att-href="url">
-                        <img src="/web/image/website.s_product_list_default_image_1" alt="" class="img img-fluid"/>
+                    <a t-att-href="url" class="text-decoration-none">
+                        <div class="s_card o_card_img_top card o_cc o_cc1" data-name="Card" data-vcss="001">
+                            <figure class="o_card_img_wrapper ratio ratio-4x3 mb-0">
+                                <img src="/web/image/website.s_product_list_default_image_1" alt="" class="o_card_img card-img-top"/>
+                            </figure>
+                            <div class="card-body text-center">
+                                <h6 class="card-title m-0">Elegant</h6>
```

Comment:
Same as my previous comment, it prevents the margin customization available in the WB.
```suggestion
                                <h6 class="card-title">Elegant</h6>
```
Same for the other cards